tests/config/global_app/test_global_app_config.py

```diff
@@ -63,8 +63,33 @@ def test_block_update_global_app_config():
     with pytest.raises(ConfigurationUpdateBlocked):
         Config.configure_global_app(root_folder="./new_root_folder/", storage_folder=".new_storage/")
 
+    with pytest.raises(ConfigurationUpdateBlocked):
+        Config.configure_global_app(root_folder="./new_root_folder/", storage_folder=".new_storage/")
+
+    with pytest.raises(ConfigurationUpdateBlocked):
+        Config.global_config.root_folder = "./new_root_folder/"
+
+    with pytest.raises(ConfigurationUpdateBlocked):
+        Config.global_config.storage_folder = ".new_storage/"
+
+    with pytest.raises(ConfigurationUpdateBlocked):
+        Config.global_config.clean_entities_enabled = True
+
+    with pytest.raises(ConfigurationUpdateBlocked):
+        Config.global_config.repository_type = "mongo"
+
+    with pytest.raises(ConfigurationUpdateBlocked):
+        Config.global_config.repository_properties = {}
+
+    with pytest.raises(ConfigurationUpdateBlocked):
+        Config.global_config.properties = {"foo": "bar"}
+
```

Comment:
MAybe we can also test the modification of an element of the property dictionary:

```suggestion
    with pytest.raises(ConfigurationUpdateBlocked):
        Config.global_config.properties = {"foo": "bar"}

    with pytest.raises(ConfigurationUpdateBlocked):
        Config.global_config.properties["foo"] = "bar"
        
```

Reply:
About this case, it will not raise the exception, but it will not work. I mean, the `properties` stays the same.

We only have a setter for the whole `properties`, not each element of the dictionary though. Any thought on this?

Reply:
My proposed is to make all property that doesn't have a setter to private property. 
However, in the scope of this ticket, do we have to go to that extreme?

Reply:
This is more than fine for now!!!!

Excellent.